### PR TITLE
moving from cloud-run branch to develop branch

### PR DESCRIPTION
**Pull request by BenGriffith**

### CODE REVIEW

File: src/cloud_run/tables/dim_tollway.py [hunk 1]
```diff
@@ -1,48 +1,17 @@
...
-        )
-        raise
-    except Exception as e:
-        tollway_logger.error(
-            "An unexpected error occurred while inserting tollway "
-            f"{message_data.tollway_name} into BigQuery: {e}",
-            exc_info=True,
-        )
-        raise
+    table_manager = TableManager(client=bigquery_client, table=TABLES["dim_tollway"], row=dim_tollway_row)
```
Comment (bbelderbos) [inline src/cloud_run/tables/dim_tollway.py:14]: this and next module's code seems similar to the previous one?

File: src/cloud_run/tables/table_manager.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        table = f"{PROJECT_ID}.{DATASET_ID}.{self.table}"
+        query = f"""
+            SELECT COUNT(*) AS record_count
+            FROM `{table}`
+            WHERE {self.key_field} = {key}
+        """
+        query_result = self.bq_client.query(query).result()
+        record_count = next(query_result).get("record_count", 0)
+
+        if record_count > 0:
```
Comment (bbelderbos) [inline src/cloud_run/tables/table_manager.py:26]: can do one liner here: `return record_count == 0` (assuming negative value would not happen)

File: src/cloud_run/tables/dim_tollway.py (hunk 1 above)
Comment (BenGriffith) [reply src/cloud_run/tables/dim_tollway.py:14]: yes, planning to clean this up in next few days. I originally broke out each table into it's own module with main difference being table row definition. Then in `main.py` only reference the each table module `insert` property/method. However, there was still quite a bit of code being repeated.

File: src/cloud_run/tables/table_manager.py (hunk 2 above)
Comment (BenGriffith) [reply src/cloud_run/tables/table_manager.py:26]: brilliant

File: src/cloud_run/tables/dim_tollway.py (hunk 1 above)
Comment (bbelderbos) [reply src/cloud_run/tables/dim_tollway.py:14]: Sounds good, sometimes we have to in order to figure things out, but now that the duplication is there you can refactor and use the extract method refactoring.

**Pull request merged**